src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr

```diff
@@ -39,9 +39,9 @@ folder InstallDir:\MSBuild\15.0\Bin\Roslyn
 
     file source=$(OutputPath)\Vsix\CompilerExtension\System.AppContext.dll vs.file.ngenArchitecture=all
     file source=$(OutputPath)\Vsix\CompilerExtension\System.Console.dll vs.file.ngenArchitecture=all
+    file source=$(OutputPath)\Vsix\CompilerExtension\System.Diagnostics.FileVersionInfo.dll vs.file.ngenArchitecture=all
     file source=$(OutputPath)\Vsix\CompilerExtension\System.Diagnostics.Process.dll vs.file.ngenArchitecture=all
     file source=$(OutputPath)\Vsix\CompilerExtension\System.Diagnostics.StackTrace.dll vs.file.ngenArchitecture=all
-    file source=$(OutputPath)\Vsix\CompilerExtension\System.Diagnostics.FileVersionInfo.dll vs.file.ngenArchitecture=all
```

Comment:
Keeping them in alphabetical order makes doing a side-by-side comparison of the file list significantly easier. Otherwise I have to do an `O(n^2)` comparison algorithm in my head. :)